css-flexbox/display-flex.html

```diff
@@ -0,0 +1,60 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="X-UA-Compatible" content="ie=edge" />
+    <title>CSS Flexbox - display: flex</title>
+    <link rel="stylesheet" href="reset.css" />
+    <link rel="stylesheet" href="pokemon.css" />
+    <style>
+      /* your styles here 👇 */
+      /* A new a new css rule set selecting all elements in the 'card-deck' id ... Opening curly bracket*/
+      #card-deck {
+        display: flex;
+        flex-direction: flex;
+        flex: row;
```

Comment:
I don't believe this exists. `flex` expects numerical values or `initial, auto, none`. I would remove it.